scripts/install_third_party.sh

```diff
@@ -172,6 +172,9 @@ if [ ! -d "$TOOLS_DIR/selenium-2.53.2" ]; then
   pip install selenium==2.53.2 --target="$TOOLS_DIR/selenium-2.53.2"
 fi
 
+pip install codecov
```

Comment:
I had a few points here: 
1. Only those modules should be placed in third party script which are required to develop and run oppia locally. Placing these here means everytime a developer runs this script, both codecov and pyyaml will be installed locally. These are not required to run the server though and will also lead to extra time for local setup.
2. This is not the right way to "pip install" any package. We specify every detail -- the exact version, directory, a verbose message that this package is currently being installed etc. See other "pip install" statements for reference.

Reply:
I was thinking about this too when I moved the commands. Since codecov is being used only for  .travis.yml. Would not make a lot of sense, to put in install_third_party.sh
`if [ $RUN_BACKEND_TESTS == 'true' ] && [ $REPORT_BACKEND_COVERAGE == 'true' ]; then codecov; fi`
However, in the .travis.yml
```bash
before_install:
...
- pip install codecov
- pip install pyyaml
```
These are ran before each Travis script, putting these packages directly into Travis's pip 's `$HOME/.cache/pip$HOME/`. I am ok without relocating the pip install commands but pip cache should be necessary here.

Reply:
Okay, so here is what I suggest:
1. Use `cache: pip`
2. Let codecov and pyyaml be there at the same place in travis.yml.
Does this sound good?
Thanks!

Reply:
Yes! Glad we are on the same page here. Reverting changes to install_third_party.sh.